pkg/gofr/gofr.go

```diff
@@ -224,6 +237,33 @@ func (a *App) Run() {
 	wg.Wait()
 }
 
+func (a *App) pingGoFr(client *http.Client, s string) {
+	url := fmt.Sprint(gofrHost, shutServerPing)
+
+	if s == "start" {
+		url = fmt.Sprint(gofrHost, startServerPing)
+
+		a.container.Info("GoFr tracks running servers via telemetry. Set GOFR_TELEMETRY=false in configs to disable it.")
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), pingTimeout)
```

Comment:
```suggestion
func (a *App) pingGoFr(ctx context.Context, client *http.Client, s string) {
	url := fmt.Sprint(gofrHost, shutServerPing)

	if s == "start" {
		url = fmt.Sprint(gofrHost, startServerPing)

		a.container.Info("GoFr tracks running servers via telemetry. Set GOFR_TELEMETRY=false in configs to disable it.")
	}

	ctx, cancel := context.WithTimeout(context.WithoutCancel(), pingTimeout)
```

You should receive the context from the caller, that's the way to do

Reply:
It is an independent call, does not need to have access to the existing context.